src/CfSshWrapper/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("CfSshWrapper")]
+[assembly: AssemblyDescription("")]
```

Comment:
```suggestion
[assembly: AssemblyDescription("Wrapper around cf ssh command to enable setting CF_HOME")]
```

Reply:
Ah yeah, nice callout -- I saw this go in, didn't have a firm opinion of what to put in there, then forgot all about it 😄